sources/packages/backend/libs/integrations/src/cas/_tests_/unit/cas.service.createSiteForExistingSupplier.spec.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  CASService,
+  CreateExistingSupplierSiteData,
+} from "@sims/integrations/cas";
+import { Mocked } from "@suites/unit";
+import { HttpService } from "@nestjs/axios";
+import {
+  DEFAULT_CAS_AXIOS_AUTH_HEADER,
+  initializeService,
+  mockAuthenticationResponseOnce,
+} from "./cas-test.utils";
+
+describe("CASService-createSiteForExistingSupplier", () => {
+  let casService: CASService;
+  let httpService: Mocked<HttpService>;
+
+  beforeAll(async () => {
+    [casService, httpService] = await initializeService();
+  });
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it("Should invoke CAS API createSiteForExistingSupplier with formatted payload when all data was provided as expected.", async () => {
```

Comment:
Thanks for adding the unit test ❤️

Please use user friendly description. `createSiteForExistingSupplier` -> `create site for existing supplier`